ci-operator/step-registry/gcp/deprovision/buildhost/OWNERS

```diff
@@ -0,0 +1 @@
+../OWNERS
```

Comment:
How about creating a plain text file for the `OWNERS` file instead of symbolic link file, that will avoid affecting other unrelated steps OWNERS change.

Reply:
fixed!